### ISSUE TITLE
Mouse events still transmitted to the active workspace when interacting with the overview

### ISSUE THREAD

**Issue by mattcollier** [mzur/gnome-shell-wsmatrix#196]
Hello, running up-to-date Ubuntu 20.04 here.

I've setup Workspace Matrix to a 2 X 2 grid with a 0.5 thumbnail scaling factor which works well on my 21:9 ultra wide display.  This means that the overview takes up nearly 100% of the display.

When I move my mouse anywhere in the overlay, I can see that the active workspace is reacting to the mouse movements and clicks just as if the overview were not present.  I can for examples see web browsers responding to mouse-over events in the preview. I can see how this could lead to all sorts of unexpected behavior especially when there are "delete" buttons that could be inadvertently clicked.

Is this the intended behavior for Workspace Matrix?

**Comment by ebeem** [comment]
This is tracked in #60 #174 #178
The drag and drop should be working properly in the future, but right now, I think it's better to disable `select workspaces with mouse hover in popup` from the extension preferences.

Check that out and let me know if this is the behavior you are looking for.

**Comment by mattcollier** [comment]
Hello, I observed that my extension preferences panel does not include the option you referenced.  I then spent a bit trying to determine what version of the extension I am running.

I gather that this code indicates that the latest code on master only works with shell versions: 40, 41
https://github.com/mzur/gnome-shell-wsmatrix/blob/992b0d00cd5ffb0af2514910d62e89bd99edca69/wsmatrix%40martin.zurowietz.de/metadata.json#L2-L5

My metadata file, after installing from extensions.gnome.org looks like this:
```
{
  "_generated": "Generated by SweetTooth, do not edit",
  "description": "Arrange workspaces in a two dimensional grid with workspace thumbnails.",
  "gettext-domain": "wsmatrix",
  "keybindings-schema": "org.gnome.shell.extensions.wsmatrix-keybindings",
  "name": "Workspace Matrix",
  "settings-schema": "org.gnome.shell.extensions.wsmatrix-settings",
  "shell-version": [
    "3.36"
  ],
  "url": "https://github.com/mzur/gnome-shell-wsmatrix",
  "uuid": "wsmatrix@martin.zurowietz.de",
  "version": 23
}
```

Indeed, I am running Gnome 3.36.8 on Ubuntu 20.04.  Is my version of Gnome considered legacy as it relates to this extension?

**Comment by ebeem** [comment]
Yeah, we rebuilt the extension from scratch since `GNOME 40`.
I hope that it will be resolved by the time you try it out.

**Comment by ebeem** [comment]
Closed as this was fixed in `GNOME +40` and is tracked in #174
